flotta-agent.spec

```diff
@@ -45,8 +45,12 @@ getent passwd %{flotta_user} >/dev/null || useradd -g %{flotta_user} -s /sbin/no
 %post
 systemctl enable --now nftables.service
 loginctl enable-linger %{flotta_user}
-systemctl start user@$(getent passwd %{flotta_user} | cut -d: -f3).service
+systemctl start user@$(id -u %{flotta_user}).service
+
+# HACK till https://bugzilla.redhat.com/show_bug.cgi?id=2060702 is fixed
+setenforce 0
 systemctl --machine %{flotta_user}@.host --user start podman.socket
```

Comment:
O wonder if there is same selinux issue if using:
```
machinectl shell flotta@.host /usr/bin/systemctl start --user podman.socket
```

Reply:
yes, same issue.

Reply:
re-tested with 
```
machinectl shell %{flotta_user}@.host /usr/bin/systemctl enable --now --user podman.socket
```
it doesn't fail the port-script, however, it also doesn't enable the service.